iterations.r

```diff
@@ -45,25 +46,36 @@ stations_metadata_df <-
 source("functions/data_tests.r")
 test_stations_metadata(stations_metadata_df)
 
-
-### 5: Final volume query: 
-
+#### 4: Query test
 source("gql-queries/vol_qry.r")
-
-stations_metadata_df %>% 
-  filter(latestData > Sys.Date() - days(7)) %>% 
-  sample_n(1) %$% 
+GQL(
   vol_qry(
-    id = id,
-    from = to_iso8601(latestData, -4),
-    to = to_iso8601(latestData, 0)
-  ) %>% 
-  GQL(., .url = configs$vegvesen_url) %>%
-  transform_volumes() %>% 
-  ggplot(aes(x=from, y=volume)) + 
-  geom_line() + 
-  theme_classic()
-
+    id=stations_metadata_df$id[1], 
+    from=to_iso8601(stations_metadata_df$latestData[1],-4),
+    to=to_iso8601(stations_metadata_df$latestData[1],0)
+  ),
+  .url = configs$vegvesen_url
+)
 
+### 5: Final volume query: 
 
+source("gql-queries/vol_qry.r")
 
+stations_metadata_df %>%
+  filter(latestData > Sys.Date() - days(7)) %>%
+  sample_n(1) %$% {
+    vol_qry(
+      id = id,
+      from = to_iso8601(.$latestData, -4),
+      to = to_iso8601(.$latestData, 0)
+    ) %>% 
+      GQL(., .url = configs$vegvesen_url) %>%
+      transform_volumes() %>%
+      ggplot(aes(x = from, y = volume)) +
+      geom_line() +
+      theme_classic() +
+      ggtitle("Name:", name)+
+      labs(x = "Date", y = "Traffic")+
+      geom_point(aes(color=volume))+
+      theme(minimal,legend.position = "none")
+  }
```

Comment:
`theme(minimal,legend.position = "none")` does not run for me. When I changed it to ` theme_minimal() + theme(legend.position = "none")` it looks great. 

Make sure that the code runs before submitting. If it works for you it could be due to different versions of R.